install_script_op_worker2.sh

```diff
@@ -164,6 +164,64 @@ if [ -n "$DD_INSTALL_ONLY" ]; then
     no_start=true
 fi
 
+# sources
+source_splunk_hec_address=
+if [ -n "$DD_OP_SOURCE_SPLUNK_HEC_ADDRESS" ]; then
+    source_splunk_hec_address=$DD_OP_SOURCE_SPLUNK_HEC_ADDRESS
+fi
+
+source_splunk_tcp_address=
+if [ -n "$DD_OP_SOURCE_SPLUNK_TCP_ADDRESS" ]; then
+    source_splunk_tcp_address=$DD_OP_SOURCE_SPLUNK_TCP_ADDRESS
+fi
+
+source_datadog_agent_address=
+if [ -n "$DD_OP_SOURCE_DATADOG_AGENT_ADDRESS" ]; then
+    source_datadog_agent_address=$DD_OP_SOURCE_DATADOG_AGENT_ADDRESS
+fi
+
+source_sumo_logic_address=
+if [ -n "$DD_OP_SOURCE_SUMO_LOGIC_ADDRESS" ]; then
+    source_sumo_logic_address=$DD_OP_SOURCE_SUMO_LOGIC_ADDRESS
+fi
+
+# source key passes
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Just confirming these names are correct because KEY_PASS sounds funny.

Reply:
yep its a bit weird but those ones are correct

Reply:
It represents the password used to encrypt the TLS secret key data.